### PR TITLE
Issue 147 update headers text

### PR DESCRIPTION
**Pull request by TheAyeshaSiddiqa**
Update text fields in Training Provider Catalogue API
image

### CODE REVIEW

**Comment by TheAyeshaSiddiqa** [comment]
@monkeypants had a question about this API. There were separate issues for each filter. But I think they are not required as all filters are being covered in a single get method of search courses. what do you say?

File: app/api/courses.py [hunk 1]
```diff
@@ -29,7 +25,28 @@ def search_course(
     availability: Optional[bool] = None,
 ):
     """
-    Search Courses using filters
+    <p>
+    This is a public, read-only API that allows Aged Care Provider (employer) to:\n
+    the employer is able to:
```
Comment (BarisSari) [inline app/api/courses.py:30]: I don't think we need this line.

File: app/api/courses.py [hunk 2]
```diff
@@ -29,7 +25,28 @@ def search_course(
...
+    <ul>
+      <li>search for courses meeting various criteria, so they can organise training for staff.</li>
+      <li>see what industry standards the training provider supports</li>
+    </ul>
+    \n
+    It is part of the interface standard because it allows employers to\n
+    federate search results from multiple training providers.\n
```
Comment (BarisSari) [inline app/api/courses.py:37]: I think you can remove `\n` from these two lines. It makes it difficult to read.

**Review by BarisSari — CHANGES_REQUESTED** [review]
I asked for only minor changes, @TheAyeshaSiddiqa. I think there is also a conflict. Can you solve it?

File: app/api/courses.py (hunk 2 above)
Comment (TheAyeshaSiddiqa) [reply app/api/courses.py:37]: I put it there because somehow list was making the font size of following text smaller, it helped to prevent that.

Comment (BarisSari) [reply app/api/courses.py:37]: Aah, okay. It's weird.

**Pull request closed without merging**